Miniscript-Template-002-5 Key Timelock Multisig.mediawiki

```diff
@@ -10,7 +10,7 @@
 
 =Goal to be achieved by template=
 
-Expanding upon the use of the 2 of 3 multisig, a 3 of 5 multisig allows for multiple timelocks to be introduced for additional flexibility. 
+Expanding upon the use of the concept of a 3 key timed multisig, a 5 key timed multisig allows for multiple timelocks to be introduced for additional flexibility. 
```

Comment:
In c2cc1af818d4a64e8de4d09794e5b6a4449964f4:

I think you meant to write "Expanding upon the concept" rather than "Expanding upon the use of the concept"

Reply:
Updated